app/models/payment_intent.rb

```diff
@@ -11,8 +11,9 @@ class PaymentIntent < ApplicationRecord
   validates :reference, :state, length: { maximum: 255 }
   validates :reference, uniqueness: true
 
-  scope :latest, -> (count = 1) { reorder(created_at: :desc, id: :desc).limit(count) }
-  scope :latest_pending, -> (count = 1) { where.not(state: SUCCEEDED_STATES).latest(count) }
+  scope :latest, ->(count = 1) { reorder(created_at: :desc, id: :desc).limit(count) }
+  scope :pending, ->() { where.not(state: SUCCEEDED_STATES) }
+  scope :latest_pending, ->(count = 1) { pending.latest(count) }
```

Comment:
I do not think we really need this scope because we can just do `latest(count).pending`